internal/scheme/migrate.go

```diff
@@ -12,6 +12,48 @@ var migrations = []darwin.Migration{
 		Description: "Create uuid extension",
 		Script:      `CREATE EXTENSION "uuid-ossp";`,
 	},
+	{
+		Version:     2,
+		Description: "Create posts table",
+		Script: `
+			CREATE TABLE posts (
+				id uuid NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+				subject_class_id uuid NOT NULL,
+				topic_subject_id uuid NOT NULL,
+				type char(5) NOT Null,
+				type_id uuid,
+				title varchar(45) NOT NULL,
+				description varchar(255),
+				file_type char(1),
+				storage_id uuid,
+				source varchar(128),
+				is_allow_to_comment bool NOT NULL,
+				updated_at timestamptz NOT NULL,
```

Comment:
sepertinya perlu ditambhakan field is_publish bool NOT NULL DEFAULT true untuk fitur publish dan unpublished post.